ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource

```diff
@@ -44,6 +45,11 @@ Open Data Science Project Details Page
         Click Link    ${project_title}
         Wait Until Project Is Open    project_title=${project_title}
     END
+    IF  "${tab_id}" != "${NONE}"
+        Click Button    //button[@role="tab" and @aria-controls="${tab_id}"]
+        Element Should Be Visible   //button[@role="tab" and @aria-controls="${tab_id}" and @aria-selected="true"]
+        Sleep    1s    reason=The tab page needs to load content according to user permissions
```

Comment:
Please, add the discussed https://issues.redhat.com/browse/RHOAIENG-4980 for reference here - once it is fixed, the performance should be improved and so we should update it here too, I think?

Hm, thinking about it more - wouldn't be better to use something like `Wait Until Page Contains`? With will wait just the right amount of time and no need to reference the issue above since we won't need to amend this code?

Reply:
I thought about mentioning it's a workaround for RHOAIENG-4980, however, quite often we already use sleep when expecting for backend response (e.g. look for `Sleep    5s    reason=let's not overload the API`). In this case we merely wait just 1 second, so it's legitimate I suppose.

I also prefer to wait for a certain element rather than static wait, but there is no such common element that appears after the tab is selected (`//button[@role="tab" and @aria-controls="${tab_id}" and @aria-selected="true"]`).
I asked Dev to add something like `//div[@aria-controls="details-section" and @aria-selected="false"]` which turns to `true` when detais section is ready.